lib/Dialect/FIRRTL/LowerToRTL.cpp

```diff
@@ -97,6 +97,29 @@ static LogicalResult lower(firrtl::ConstantOp op, ArrayRef<Value> operands,
   return success();
 }
 
+static LogicalResult lower(firrtl::WireOp op, ArrayRef<Value> operands,
+                           ConversionPatternRewriter &rewriter) {
+  auto resType = op.result().getType();
+  if (auto resFirType = resType.dyn_cast<FIRRTLType>()) {
+    auto resultType = RTLTypeConverter::convertType(resFirType);
+
+    if (auto intType = resultType.getValue().dyn_cast<IntegerType>()) {
+      rewriter.replaceOpWithNewOp<rtl::WireOp>(op, intType, op.nameAttr());
+    }
+    // TODO: Add support for vectors in RTL. The below code to handle FIRRTL
```

Comment:
nit: Can you move the comment here so that the control flow is obvious? The way this is generally done is something like:

if (...) {

  // My comment goes here.
} else if (...) {

}